README.md

```diff
@@ -62,4 +62,53 @@ This will start a local development server on port 5000 (http://localhost:5000).
 This project welcomes contributors. If you are interested, please feel free to
 join [the mailing list](https://mail.mozilla.org/listinfo/bugsahoy-devel)
 
+## Add a project
+
+Each of the projects is a file in the [data](https://github.com/mozilla-frontend-infra/codetribute/tree/master/src/data)
+folder - to add a new one, create a new file named after the project, ending in `.yaml`. Ensure all spaces and special
+characters are replaced with `-`. Make sure too that the project hasn't already been inside the [data](https://github.com/mozilla-frontend-infra/codetribute/tree/master/src/data)
+folder
+
+The contents of the file are just some details about the project:
```

Comment:
How about:

Codetribute can read from both GitHub and Bugzilla. To add a new entry to the site, create a file `<project-name>.yml` in `src/data` using the [template example](#template-example) as the initial setup. For inspiration, check out the [Taskcluster](https://github.com/mozilla-frontend-infra/codetribute/blob/master/src/data/taskcluster.yaml) yaml file.